app/services/carrierwave_filename.rb

```diff
@@ -1,9 +1,18 @@
-module CarrierwaveFilename
```

Comment:
lolz, I thought thats what I'd pushed - well spotted. was playing around with module_function and class separately.. & theres me adding a review label & not even looking over it myself. :boom: :gun: :skull: